phonebook.js

```diff
@@ -1,22 +1,22 @@
-var readline = require('readline');
-var fs = require('fs');
-var promisify = require('util').promisify
+const readline = require('readline');
```

Comment:
I like the use of const on these lines.

Reply:
@leecmoses I like your food

Reply:
I have plenty, would you like some for lunch?

Reply:
I have plenty of food too, but what exactly are you willing to share?